code/app/src/main/java/com/example/the_tarlords/ui/event/EventDetailsFragment.java

```diff
@@ -236,4 +247,15 @@ else if(menuItem.getItemId()==R.id.mapOptionsMenu) {
 
     }
 
+    /**
```

Comment:
Is this function currently called anywhere or does it need to be implemented? Also I'm assuming we're doing a blanket notif permission for the whole app and not per event? doing it per event seems complicated